pontoon/base/forms.py

```diff
@@ -91,6 +92,8 @@ class UserPermissionLogFormMixin:
     def __init__(self, *args, **kwargs):
         self.user = kwargs.pop("user")
         super().__init__(*args, **kwargs)
+        # Track if user reached new level for Community Builder Badge
+        self.community_builder_level = 0
```

Comment:
If we're only setting this variable when the new level has been reached, we should reflect that in the variable name, otherwise it becomes confusing, especially on frontend.

For example:
```suggestion
        self.new_community_builder_level = 0
```

Or:
```suggestion
        self.community_builder_level_reached = 0
```